test/behave/02_turn_on_switch.feature

```diff
@@ -0,0 +1,5 @@
+Feature: turn on
+  Scenario: turn on switch
+    Given an English speaking user
+    When the user says "can you turn on Mycroft sw please"
+	  Then "homeassistant" should reply with dialog from "homeassistant.device.on.dialog"
```

Comment:
Understood, that is reasonable. I will try to add function to set entity to default state so if one test fail, other will still run. Otherwise I will have to create separate entity for each test, doable, but not so convenient.